src/Test/Perf/util/test_util.csx

```diff
@@ -81,6 +81,11 @@ string BinDirectory()
     return Path.Combine(RoslynDirectory(), "Binaries");
 }
 
+string PerfTestDirectory()
```

Comment:
No real use making a helper method if it's only going to be called by the runner.  (even then, it should probably be in `runner_util.csx`.